proxy-node-shared/src/main/java/uk/gov/ida/notification/shared/logging/ProxyNodeLoggingFilter.java

```diff
@@ -29,25 +30,24 @@
 
     @Override
     public void filter(ContainerRequestContext requestContext) {
-        String istioTraceId = requestContext.getHeaderString(X_B3_TRACEID);
-        if (istioTraceId != null && !istioTraceId.isBlank()) {
-            MDC.put(X_B3_TRACEID, istioTraceId);
-        }
+        ofNullable(requestContext.getHeaderString(X_B3_TRACEID))
+            .filter(Predicate.not(String::isBlank)).ifPresent(traceId -> MDC.put(X_B3_TRACEID, traceId));
```

Comment:
Complete nitpick, but the `Predicate.not` method is a fully-qualified import, whilst the rest of the line uses static imports. Can we make it a static import?